renderdoc/serialise/serialiser.h

```diff
@@ -592,7 +592,7 @@ class Serialiser
     }
 
     char *data = (char *)ReadBytes(sizeof(T));
-    f = *((T *)data);
+    memcpy(&f, data, sizeof(T));
```

Comment:
Its not specific to any type. Data fetches on Arm have to be aligned according to the size of the type, therefore 64bit values must be 8 byte aligned etc.